config/manifests/bases/kubernetes-image-puller-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,64 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments: null
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - che
+  - kubernetes-image-puller
+  labels:
+    app: kubernetes-image-puller-operator
+  links:
+  - name: Repository
+    url: https://github.com/che-incubator/kubernetes-image-puller-operator
+  maintainers:
+  - email: ibuziuk@redhat.com
+    name: Ilya Buziuk
+  - email: tgeorge@redhat.com
+    name: Tom George
+  maturity: alpha
+  provider:
+    name: Red Hat, Inc.
+    url: https://github.com/che-incubator/kubernetes-image-puller-operator
+  replaces: kubernetes-imagepuller-operator.v0.0.8
+  selector:
+    matchLabels:
+      app: kubernetes-image-puller-operator
+  version: 0.0.0
```

Comment:
I think this should be `0.0.9`

Reply:
The same This file, in general is a template.